text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,41 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+Clarify API description namespace and iron out issues which rose during API Blueprint serialization to API description namespace.
+
+# Motivation
+The purpose of this document is to clear inconsistencies in the format and make it easier to build tools around it.
+
+# Detailed design
+
+## Data Structure
+Since `Data Structure` inheriting from `Data Structure Element` of DataStructure Correct the `Data Structure` to the following:
+
+```apib
+# Data Structure (Element)
++ element: dataStructure (string, fixed)
++ content (Data Structure Element)
+```
+
+## Transition Example
+Value of `content` should be changed to `[]` instead of `null`.
+
+## Category
+Remove the "reserved" word from the description of `scenario` since it can be used as category semantics.
+
+## Copy
+We need to clarify what the relation between copy elements in `content` and `description` in `attributes` is. Unless specified otherwise, a copy element's content should represent the description of it's parent element and SHOULD be used instead of parent element's description metadata. `Copy Element` should be added to `content` of the following elements in API namespace:
+
+- Category Element
+- Resource Element
+- Transition Element
+- HTTP Transaction Element
+- HTTP Message Payload Element
+
+## HTTP Transaction Example
+Value of `content` should be changed to `[]` instead of `null`.
+
+## HTTP Headers
+We need to correct the `content` to be an `array[Member Element]`.
```

Comment:
Stylistics: 

Seems you have two "voices" in this document:

1. X should be changed to Y
2. We need X to be Y 

While I get what you are trying to say it feels like one is a recommendation while the other is a must.